domain/model/health.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+import "time"
+
+/*
+	Health - Contains the following fields:
+	- Uptime: Uptime of project since last build in ms.
+	- StatusCode: HTTPStatus.
+*/
+type Health struct {
```

Comment:
Comments that describe exported elements should be written as single-line comments by convention; please refer to https://go.dev/blog/godoc and fix all other occurrences of this specific issue.

```suggestion
// Health - Contains the following fields:
//	Uptime: Uptime of project since last build in ms.
//	StatusCode: HTTPStatus.
type Health struct {
```
Or something similar to this suggestion, just make sure all of these are single-line comments.